internal/errors/mysql_test.go

```diff
@@ -177,7 +521,6 @@ func TestErrMySQLNotFoundIdentity_Unwrap(t *testing.T) {
 }
 
 func TestIsErrMySQLNotFound(t *testing.T) {
-	t.Parallel()
```

Comment:
this code occurs goroutine detect error like this.
I think there are other ways. However, I can't find it at the moment, so I'll try another PR.
Because this PR period is getting longer.
```
 Goroutine 22 in state sleep, with time.Sleep on top of the stack:
        goroutine 22 [sleep]:
        time.Sleep(0x5f5e100)
        	/usr/local/go/src/runtime/time.go:188 +0xbf
        go.uber.org/goleak.(*opts).retry(0xc0001260f0, 0x13, 0x8)
        	/go/pkg/mod/go.uber.org/goleak@v1.1.10/options.go:119 +0x65
        go.uber.org/goleak.Find(0x0, 0x0, 0x0, 0xc000108040, 0x3b)
        	/go/pkg/mod/go.uber.org/goleak@v1.1.10/leaks.go:66 +0x134
        go.uber.org/goleak.VerifyNone(0x67b3c0, 0xc000083080, 0x0, 0x0, 0x0)
        	/go/pkg/mod/go.uber.org/goleak@v1.1.10/leaks.go:77 +0x45
        github.com/vdaas/vald/internal/errors.TestIsErrMySQLInvalidArgument.func3.1(0xc000083080)
        	/go/src/github.com/vdaas/vald/internal/errors/mysql_test.go:789 +0xb5
        github.com/vdaas/vald/internal/errors.TestIsErrMySQLInvalidArgument.func3(0xc000083080)
        	/go/src/github.com/vdaas/vald/internal/errors/mysql_test.go:809 +0x2a9
        testing.tRunner(0xc000083080, 0xc0000ba0e0)
        	/usr/local/go/src/testing/testing.go:1108 +0xef
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1159 +0x386

```